geth/log/log.go

```diff
@@ -78,11 +81,13 @@ func SetLevel(level string) {
 // SetLogFile configures logger to write output into file.
 // This call preserves current logging level.
 func SetLogFile(filename string) error {
-	handler, err := log.FileHandler(filename, log.TerminalFormat(false))
-	if err != nil {
-		return err
+	logger.ljWriter = &lumberjack.Logger{
```

Comment:
How does `lubmerjack` report problems with e.g. creating a file? We used to have some issues with creating logs files via status-go, thus we have this code: https://github.com/status-im/status-react/blob/6d5251ad2072c949c0608acde4144d4f4cff0d11/modules/react-native-status/ios/RCTStatus/RCTStatus.m#L221-L226. 

I will try to test a case when a new log file is created before merging this PR.

Reply:
`lumberjack` errors will not be reported, since the geth logger discards the return value from `io.Writer`. There's no reason why keeping the code you mentioned wouldn't work though, unless I'm missing something. The code you mentioned is meant to address the scenario where a user/something changes the permissions on a log file, right?

By the way, noticed that I used v1 of `lumberjack`, but a v2 is available and recommended, so I'll update that.

Reply:
> There's no reason why keeping the code you mentioned wouldn't work though, unless I'm missing something. The code you mentioned is meant to address the scenario where a user/something changes the permissions on a log file, right?

iOS is a very specific platform. I don't remember what exactly was the problem but ultimately the log file needed to be created in Objective-C before status-go started using it. We just need to test it as a whole and figure out a better solution if there will be a problem.

> By the way, noticed that I used v1 of lumberjack, but a v2 is available and recommended, so I'll update that.

👍